zerver/tests/test_events.py

```diff
@@ -2028,6 +2028,9 @@ def test_do_delete_message(self) -> None:
             ('type', equals('delete_message')),
             ('message_id', check_int),
             ('sender', check_string),
+            ('is_stream_message', check_bool),
+            ('stream_id', check_int),
+            ('subject', check_string),
         ])
```

Comment:
We should split this into two tests, one for stream and one for private, to handle the cases above.